### PR TITLE
Clarify how functions are uploaded with wrangler pages

### PR DESCRIPTION
**Pull request by IanVS**
I was recently confused how to upload functions using wrangler.  Hopefully these changes can help make it clear to others.

### CODE REVIEW

**Review by jasikpark — COMMENTED** [review]
lgtm

File: content/pages/platform/direct-upload.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ If using the Drag and Drop method, a red warning symbol will appear next to an a
 
 ### Using Functions
 
-Drag-and-drop deployments made from the Cloudflare dashboard do not currently support compiling a `functions` folder of Pages Functions. In order to deploy a `functions` folder, you must use Wrangler.
+Drag-and-drop deployments made from the Cloudflare dashboard do not currently support compiling a `functions` folder of Pages Functions. In order to deploy a `functions` folder, you must use Wrangler, which will automatically upload a `functions` folder from the directory where it is run.
```
Comment (WalshyDev) [inline content/pages/platform/direct-upload.md]:
```suggestion
Drag-and-drop deployments made from the Cloudflare dashboard do not currently support compiling a `functions` folder of Pages Functions.
In order to deploy a functions folder, you must use Wrangler. When deploying a project using Wrangler, if a `functions` folder exists where the command is run, that will be uploaded with the project.
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -1257,6 +1257,12 @@ Your site is deployed to `<PROJECT_NAME>.pages.dev`. If you do not provide the `
 
 {{<Aside type="note">}}
 
+Functions in a `functions` folder at the CWD will also be uploaded.
+
+{{</Aside>}}
```
Comment (WalshyDev) [inline content/workers/wrangler/commands.md]: I'm not sure this brings any benefit.  I'd rather we describe this more clearly elsewhere than add _another_ aside.
```suggestion
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (IanVS) [reply content/workers/wrangler/commands.md]: Ok that's fine. I think the other spot is probably enough.

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -1255,6 +1255,7 @@ Your site is deployed to `<PROJECT_NAME>.pages.dev`. If you do not provide the `
 
 {{</Aside>}}
 
+
```
Comment (WalshyDev) [inline content/workers/wrangler/commands.md]:
```suggestion
```

**Review by jasikpark — COMMENTED** [review]
lgtm

File: content/pages/platform/direct-upload.md [hunk 4]
```diff
@@ -125,7 +125,8 @@ If using the Drag and Drop method, a red warning symbol will appear next to an a
 
 ### Using Functions
 
-Drag-and-drop deployments made from the Cloudflare dashboard do not currently support compiling a `functions` folder of Pages Functions. In order to deploy a `functions` folder, you must use Wrangler.
+Drag-and-drop deployments made from the Cloudflare dashboard do not currently support compiling a `functions` folder of Pages Functions.
+In order to deploy a functions folder, you must use Wrangler. When deploying a project using Wrangler, if a `functions` folder exists where the command is run, that will be uploaded with the project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Drag and drop deployments made from the Cloudflare dashboard do not currently support compiling a `functions` folder of Pages Functions. To deploy a `functions` folder, you must use Wrangler. When deploying a project using Wrangler, if a `functions` folder exists where the command is run, that `functions` folder will be uploaded with the project.
```

**Review by deadlypants1973 — APPROVED** [review]

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]
Added wording for proper noun clarity

**Review by deadlypants1973 — APPROVED** [review]

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**